src/osd/osd_types.cc

```diff
@@ -2363,6 +2364,7 @@ void pg_stat_t::encode(bufferlist &bl) const
   ::encode(last_peered, bl);
   ::encode(last_became_peered, bl);
   ::encode(pin_stats_invalid, bl);
+  ::encode(snaptrimq_len, bl);
```

Comment:
this is going to cause issues interacting with master, since it already has different fields added as encoding version 23 and 24. Is it feasible to backport those encodings or use dummy values for them, so snaptrimq_len is version 25 here like in master?

Reply:
@jdurgin That's why I made https://github.com/ceph/ceph/pull/19965 that's already merged.

Reply:
ah excellent, I was going through these too quickly and missed that. Thanks!